.gitmodules

```diff
@@ -3,5 +3,5 @@
 	url = https://github.com/ibayer/fastFM-core.git
 [submodule "fastFM-core2"]
 	path = fastFM-core2
-	url = git@github.com:ibayer/fastFM-core2.git
+	url = git@github.com:AlexJoz/fastFM-core2.git
```

Comment:
Probably need to be changed back or?

Reply:
yep, i'll fix it back, after i can test with your repo